tests/benchmarks/multicore/idle_twim/CMakeLists.txt

```
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+if (NOT SYSBUILD)
```

```suggestion
if(NOT SYSBUILD)
```
No spaces between these in cmake